io.asgardio.dotnet.oidc.sdk.sample/App.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿/* 
```

Comment:
License header is wrong. Year and the formatting.

Reply:
Formatting is resolved in https://github.com/wso2-extensions/identity-sdks-dotnet/pull/6/commits/ad95c5751c2becdbce174d985942960300fed9bc.
However, the didn't change the year as this was a file created when migrating the Dotnet SDK sample from https://github.com/wso2-extensions/identity-samples-dotnet